platformio.ini

```diff
@@ -16,6 +16,11 @@ default_envs = lolin_d32_pro_sdmmc_pe
 board_build.flash_mode = qio
 board_build.bootloader = dio
 board_build.partitions = custom_4mb_noota.csv
+
+; performance changes, enable these to make the esp faster (also expect higher power usage)
+board_build.f_cpu = 240000000L
```

Comment:
All the board configurations we currently support already have `f_cpu` set to 240 MHz  by default. What's the purpose of duplicating that here?

Reply:
Funnily I have not seen, that there is a whole list of boards and their default CPU speed. I'll update the comment and comment it out so that users can activate it, if they want to.